client/stylesheets/_toCat.scss

```diff
@@ -2774,6 +2774,10 @@ i:hover {
     margin: 11px 2px 7px 2px !important;
 }
 
+.headerNameOptions a:hover, .headerNameOptions a:active, .headerNameOptions a:visited, .headerNameOptions a:link {
+    color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces